### PR TITLE
fix:Added the following changes:

### PR DESCRIPTION
**Pull request by ravali-rimmalapudi**
1. Updated OAI commit not to include in the release notes
2. Updated github dependencies
3. Eslint fixes

# Fixes #

Closes: [DII-133](https://issues.corp.twilio.com/browse/DII-133)
Dependent PR: [CLI](https://github.com/twilio/twilio-cli/pull/301
)
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli-core/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

**Comment by ravali-rimmalapudi** [comment]
Successful Runs:
Cli-core: https://github.com/ravali-rimmalapudi/twilio-cli-core/actions/runs/1294686886
Release tag: https://github.com/ravali-rimmalapudi/twilio-cli-core/releases/tag/5.29.1

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -77,9 +77,11 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
```
Comment (shrutiburman) [inline .github/workflows/release.yml:77]: [nit] : If `GITHUB_TOKEN` isn't used anymore, can we remove it? Same with the related PR too.

Comment (ravali-rimmalapudi) [reply .github/workflows/release.yml:77]: It will be used by `semantic-release`  to push a new tag while doing the Github release.

Comment (shrutiburman) [reply .github/workflows/release.yml:77]: I see.

**Review by shrutiburman — DISMISSED** [review]
🚀

File: .github/scripts/trigger-workflow.js [hunk 2]
```diff
@@ -8,45 +8,39 @@ const { Octokit } = require('@octokit/rest');
...
     const dispatchResp = await octokit.rest.actions.createWorkflowDispatch({
       owner,
       repo,
       workflow_id: workflow.data.id,
       ref,
-      inputs
+      inputs,
     });
-    core.info(`API response status: ${dispatchResp.status} 🚀`)
+    core.info(`API response status: ${dispatchResp.status} 🚀`);
```
Comment (shamantraghav) [inline .github/scripts/trigger-workflow.js:39]: You need to close the quotes. Also, what's the story behind the emoji?

Comment (shrutiburman) [reply .github/scripts/trigger-workflow.js:39]: I think the rocket's nose cone is hiding the `

Comment (ravali-rimmalapudi) [reply .github/scripts/trigger-workflow.js:39]: Haha, yeah. There is no story :sweat_smile:, I took the emoji from [here](https://github.com/benc-uk/workflow-dispatch/blob/master/src/main.ts#L55).

Comment (shamantraghav) [reply .github/scripts/trigger-workflow.js:39]: Haha okay. But the rocket would only make sense if the status was positive

**Review by shamantraghav — APPROVED** [review]

File: .github/scripts/trigger-workflow.js (hunk 2 above)
Comment (ravali-rimmalapudi) [reply .github/scripts/trigger-workflow.js]: Yeah, that's true. Updated.

**Review by shamantraghav — APPROVED** [review]

**Review by shrutiburman — APPROVED** [review]

**Pull request merged**